extensions/aws2-s3/deployment/src/main/java/org/apache/camel/quarkus/component/aws2/s3/deployment/Aws2S3Processor.java

```diff
@@ -89,12 +89,6 @@ void client(BeanRegistrationPhaseBuildItem beanRegistrationPhase,
             BuildProducer<NativeImageProxyDefinitionBuildItem> proxyDefinition) {
         checkClasspath(APACHE_HTTP_SERVICE, "apache-client");
 
-        //Register Apache client as sync client
-        proxyDefinition.produce(
-                new NativeImageProxyDefinitionBuildItem("org.apache.http.conn.HttpClientConnectionManager",
-                        "org.apache.http.pool.ConnPoolControl",
-                        "software.amazon.awssdk.http.apache.internal.conn.Wrapped"));
-
         serviceProvider.produce(new ServiceProviderBuildItem(SdkHttpService.class.getName(), APACHE_HTTP_SERVICE));
```

Comment:
Similar as above: Isn't this serviceProvider also a good candidate to move to the new support extension?

Reply:
Actually moving it there doesn't work, at least for me. I'll have to dig a bit more in this. This is the beginning.